docs/cli/profile.md

```diff
@@ -0,0 +1,31 @@
+## ⏱️ Profiling Models
+
+`nos profile` supports model benchmarking across a number of axes including iterations per second,
+memory footprint and utilization on CPU/GPU.
+Currently, the nos profiler itself runs natively in the execution environment (i.e. outside
+the NOS server), so you'll need to install both the `server` and `test` dependencies alongside
+your existing NOS installation 
+
+```bash
+pip install torch-nos[server,test]
+```
+
```

Comment:
It might be helpful to break this README by subcommand: 

First, can you print the output of `nos profile --help` here so that the users can see the relevant subcommands supported?

Then you can have different sections such as 
## `nos profile rebuild-catalog`
...

## `nos profile method -m <model-id>`

## `nos profile method -t <model-type>`